server/src/main/java/org/jboss/as/server/ServerEnvironmentService.java

```diff
@@ -69,6 +72,9 @@ public static void addService(ServerEnvironment serverEnvironment, ServiceTarget
             .install();
     }
 
+    private static final String SERVER_ENVIRONMENT_SERVICE_NAME = "jboss.server.environment";
```

Comment:
@bstansberry I'm not sure if we are following any specific rule for the capability names that replace the service usage, I think we should use something under `org.wildfly` name space. Could you add your feedback regarding it?

Reply:
@yersan I agree. I suggest using "org.wildfly.server.environment" or similar, where the exposed ServerEnvironment is also available via its legacy ServiceName for compatibility purposes.

Reply:
Thanks for confirming it @pferraro 

> where the exposed ServerEnvironment is also available via its legacy ServiceName for compatibility purposes.

Right, we have to add an alias in this case